bats/tests/helpers/vm.bash

```diff
@@ -62,6 +62,7 @@ start_container_engine() {
         --application.updater.enabled=false
         --container-engine.name="$RD_CONTAINER_ENGINE"
         --kubernetes.enabled=false
+        --experimental.virtual-machine.mount.type="$RD_MOUNT_TYPE"
```

Comment:
Shouldn't  this argument be passed only if is_unix? If so, please move this falg to line 71